src/pages/implementations.astro

```diff
@@ -89,6 +96,35 @@ const communityServerImplementations = await getCollection(
     }
   </ul>
 
+  <h3>Clients</h3>
+
+  <ul class="implementations">
+    {
+      officialClients.map((implementation) => (
+        <li>
```

Comment:
It seems like this block is repeated the third time on this page now. Can we reduce the repetition here?